ui/src/kapacitor/components/LogsTable.js

```diff
@@ -0,0 +1,98 @@
+import React, {Component, PropTypes} from 'react'
+
+import FancyScrollbar from 'shared/components/FancyScrollbar'
+import LogItemSession from 'src/kapacitor/components/LogItemSession'
+import LogItemHTTP from 'src/kapacitor/components/LogItemHTTP'
+import LogItemHTTPError from 'src/kapacitor/components/LogItemHTTPError'
+import LogItemKapacitorPoint from 'src/kapacitor/components/LogItemKapacitorPoint'
+import LogItemKapacitorError from 'src/kapacitor/components/LogItemKapacitorError'
+import LogItemKapacitorDebug from 'src/kapacitor/components/LogItemKapacitorDebug'
+import LogItemInfluxDBDebug from 'src/kapacitor/components/LogItemInfluxDBDebug'
+
+class LogsTable extends Component {
+  constructor(props) {
+    super(props)
+  }
+
+  renderTableRow = (logItem, index) => {
```

Comment:
Let's make this an actual component.  

When we make this list filterable, we will need to remove the index as keys.  Since the list is currently static, this should be good for now :).